eng/Version.Details.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
+    <Dependency Name="Microsoft.Internal.CodeCoverage" Version="17.0.0-beta.21377.6">
```

Comment:
Why don't you use `MicrosoftInternalCodeCoverageVersion` parameter from Versions.props here?

Reply:
This version is controlled by Darc. This will be automatically changed.

Reply:
Check here: https://github.com/dotnet/arcade/blob/main/Documentation/Darc.md#updating-dependencies-in-your-local-repository

Reply:
This is example of such automatic update: https://devdiv.visualstudio.com/DevDiv/_git/vs-code-coverage/pullrequest/340431?_a=files&path=%2Feng%2FVersions.props
You can see that version is changed in both places